apps/condo/domains/acquiring/schema/RegisterMultiPaymentService.js

```diff
@@ -553,88 +556,83 @@ const RegisterMultiPaymentService = new GQLCustomSchema('RegisterMultiPaymentSer
                 }
 
                 // Stage 3 Generating payments
-                const payments = []
```

Comment:
It looks like too many changes for your task. Why are you rewrite stage 3 ? @dkoviazin could you please check this changes

Reply:
It was a copy-paste from the start. 
We have many tests for registerMultiPayment with invoices or receipts and they are all working
So, LGTM